pkg/apis/serving/fieldmask_test.go

```diff
@@ -719,7 +719,11 @@ func TestPodSecurityContextMask(t *testing.T) {
 		},
 	}
 
-	want := &corev1.PodSecurityContext{}
+	want := &corev1.PodSecurityContext{
```

Comment:
I think we may also need to set RunAsNonRoot to true here. So like:

```
want := &corev1.PodSecurityContext{
                RunAsNonRoot: ptr.Bool(true),
		SeccompProfile: &corev1.SeccompProfile{
			Type: corev1.SeccompProfileTypeRuntimeDefault,
		},
	}
```

Reply:
ah actually, this is just the test. I think that change would need to be somewhere else